<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsTab</class>
 <widget class="QWidget" name="OptionsTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="optionsGroup">
     <property name="title">
      <string>Options</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="modsGroup">
     <property name="title">
      <string>Mods</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="keybindingsGroup">
     <property name="title">
      <string>Keybindings</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Reset to Defaults</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
